radio/src/datastructs.h

```diff
@@ -643,6 +643,20 @@ PACK(struct ModelData {
 
   char modelRegistrationID[PXX2_LEN_REGISTRATION_ID];
 
+//OW
+#if defined(TELEMETRY_MAVLINK)
```

Comment:
I hope this is not too much to ask, but could you please make MAVLink activation a run-time option rather than a compilation option?

Reply:
I hear your words, but I'm frankly not sure how to do that without just spilling ram for everyone ... it would be not easy to do IMHO and probably would need major redesign of the memory usage and allocation
not sure you have seen that, but mavlink support is quite RAM hungry, and I hence had to bite into the .ram section

Reply:
After discussing it again with @rotorman, I think you're right. Your PR is quite disruptive in many aspects, so that it is probably safer to keep it as a build option for now. So, forget about that comment ;-)